tools/packaging/kata-deploy/examples/nginx-deployment-dragonball.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: apps/v1
```

Comment:
@bergwolf I have added `nginx-deployment-dragonball.ymal ` and `test-deploy-kata-dragonball.yaml`. And I tested that with my local k8s cluster, it works fine.

Reply:
Then please enable it in https://github.com/kata-containers/kata-containers/blob/main/tools/packaging/kata-deploy/action/test-kata.sh#L69

Reply:
fixed